packages/interactivity/src/directives.tsx

```diff
@@ -280,9 +286,10 @@ export default () => {
 							`The value of data-wp-context in "${ namespace }" store must be a valid stringified JSON object.`
 						);
 					}
-					updateSignals( currentValue.current, {
-						[ namespace ]: deepClone( value ),
-					} );
+					updateContext(
+						currentValue.current[ namespace ],
+						deepClone( value ) as object
```

Comment:
Uhm, if I remember correctly, I added that because the parsed JSON defined in the `data-wp-context` directive―i.e., the object returned by `toVdom()` as a prop―was being modified and that caused some sort of trouble during client-side navigation.

Maybe that won't be needed once we fix https://github.com/WordPress/gutenberg/issues/63864. 🤔

Reply:
> the object returned by toVdom() as a prop―was being modified

Hmm... by who?